compiler/src/iree/compiler/Preprocessing/Passes.cpp

```diff
@@ -137,6 +150,15 @@ void registerPreprocessingPasses() {
             buildTransposeConvolutionPassPipeline(passManager,
                                                   transformOptions);
           });
+
+  PassPipelineRegistration<TransformOptions>
+      preprocessingMakeSingleDispatchPassPipeline(
+          "iree-preprocessing-make-single-dispatch",
+          "Runs passes to get a single dispatch for a function",
```

Comment:
I'm not sure I follow why we need this pass pipeline. If it only has a single pass in it without specifying any specific pass options, isn't this just shorthand for the pass name?

Reply:
For now its a single pass pipeline. Will add more to this as the bag-of-ops work continues.